src/sipnet/sipnet.c

```diff
@@ -981,26 +735,39 @@ int pastLeafFall(void) {
                                // day
 }
 
-// calculate leafCreation and leafLitter fluxes (g C/m^2 ground/day)
-// leafCreation is a fraction of recent mean npp, plus some constant amount at
-// start of growing season leafLitter is a constant rate, plus some additional
-// fraction of leaves at end of growing season
+/*!
+ * Calculate leaf creation and leaf litter fluxes
+ *
+ * Leaf creation is a fraction of recent mean npp, plus some constant amount
```

Comment:
Please confirm this is defined correctly in the parameters.md file; code says 

```
*leafCreation += (params.leafGrowth / climate->length);
```

but docs say units are g C leaf / m-2 ground. (i.e. not a rate that needs to be divided by time step but a fixed amount)

Reply:
OK Now I think I understand. 

params.leafGrowth is a parameter that is a fixed amount of leaf that is added at the start of the season, and it is divided by the time step so that it can be tracked as a flux in a single time step.